zds/member/tests/factories.py

```diff
@@ -24,6 +24,22 @@ class Meta:
 
     is_active = True
 
+    @classmethod
+    def _after_postgeneration(cls, instance, create, results=None):
+        """
+        Save the instance again after the post generation operations.
+
+        Skip the save when the object was ‘built’ and not ‘created’. See the Factory Boy
+        documentation for details on those build strategies.
+
+        Needed to save the password which is set in a post generation method call.
+
+        This method replaces the deprecated method from DjangoModelFactory which is
+        scheduled to be removed in the next major release.
```

Comment:
```suggestion
        Save the instance again after the post generation operations. Needed to save
        the password which is set in a post generation method call.

        Skip the save when the object was ‘built’ and not ‘created’. See the Factory Boy
        documentation for details on those build strategies.

        This method replaces the deprecated method from DjangoModelFactory which is
        scheduled to be removed in the next major release (see factory_boy's changelog
        entry for version 3.3.0).
```

En lisant le commentaire original, j'avais du mal à comprendre pourquoi il y en avait besoin et d'où ça venait. Donc je propose cette reformulation (c'est surtout un changement d'ordre des phrases et quelques précisions).

(attention, si tu acceptes la proposition de modification depuis l'interface de GitHub, je suspecte que black ne soit ensuite pas content, parce que j'ai l'impression que ça a fait un peu n'importe quoi avec l'indentation du commentaire...)

Reply:
C'est fait. J'ai repris ton texte sans passer par GitHub pour éviter les surprises.